R/get_infect_mat.R

```diff
@@ -65,25 +65,28 @@ get_infect_mat <- function(alpha = 0.2, R = 2.5, kappa = 0.5, eta = 0.5, nu = 4,
     t_pa,
     offset = offset, shape = shape, rate = rate
   )
+
+  second_gen_shape <- shape + (shape / rate + t_incubation + offset)
```

Comment:
@kgrantz & @jlessler

I've updated this based on our conversation today. For second generation, the shape parameter is now:

the chosen shape for the infectiousness distribution for the first generation + (mean infectiousness dist + 5.5 - 2.31)